tests/acceptance/features/apiOcm/deleteShare.feature

```diff
@@ -0,0 +1,213 @@
+@ocm @issue-enterprise-7075
+Feature: delete OCM share
+  As a user
+  I want to delete share
+  So that I can remove user access to resource
+
+  Background:
+    Given using SharingNG
+    And using spaces DAV path
+    And user "Alice" has been created with default attributes
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And user "Brian" has been created with default attributes
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+
+
+  Scenario Outline: deleting ocm shared file should delete share on both instances (Personal Space)
+    Given user "Alice" has uploaded file with content "ocm test" to "textfile.txt"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | textfile.txt       |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    And user "Alice" has deleted the last share
+    When user "Alice" lists the shares shared by her using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+        {
+          "type": "object",
+          "required": ["value"],
+          "properties": {
+            "value": {
+              "type": "array",
+              "minItems": 0,
+              "maxItems": 0
+            }
+          }
+        }
+      """
+    And using server "REMOTE"
+    When user "Brian" lists the shares shared with him without retry using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+        {
+          "type": "object",
+          "required": ["value"],
+          "properties": {
+            "value": {
+              "type": "array",
+              "minItems": 0,
+              "maxItems": 0
+            }
+          }
+        }
+      """
+    Examples:
+      | permissions-role |
+      | Viewer           |
+      | File Editor      |
+
+
+  Scenario Outline: deleting ocm shared folder should delete share on both instances (Personal Space)
```

Comment:
please rewrite scenario because it is confusing, whether you are deleting share invitation or shared folder